airbyte-connector-builder-server/connector_builder/impl/default_api.py

```diff
@@ -23,11 +20,18 @@
 from connector_builder.generated.models.streams_list_read_streams import StreamsListReadStreams
 from connector_builder.generated.models.streams_list_request_body import StreamsListRequestBody
 from connector_builder.impl.low_code_cdk_adapter import LowCodeSourceAdapter
+from fastapi import Body, HTTPException
+from jsonschema import ValidationError
 
 
 class DefaultApiImpl(DefaultApi):
     logger = logging.getLogger("airbyte.connector-builder")
 
+    def __init__(self, adapter_cls: Type, max_record_limit: int = 1000):
```

Comment:
why are we passing the class instead of the object?

Reply:
The object requires a manifest, which we don't have when `DefaultApiImpl` is instantiated.